database-mongo/index.js

```diff
@@ -35,8 +34,10 @@ const populate = (restaurant) => {
     });
     savedData.save();
   });
+  setTimeout(() => {
+    db.close();
```

Comment:
Replace with a callback. Set timeout is no good if we have 45 million pieces of data.